### PR TITLE
ign -> gz Namespace Migration : sdformat

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/gazebo-tooling/release-tools/issues/711

### CODE REVIEW

File: Migration.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ The following deprecated methods and classes have been removed.
 1. **sdf/InterfaceElements.hh**: The struct `NestedInclude` has been converted 
    to a class. Accessing data members directly is deprecated. Instead use the 
    corresponding member functions.
+2. The `ignition` namespace is deprecated and will be removed in future versions.  Use `gz` instead.
```
Comment (chapulina) [inline Migration.md:147]: This is under the 11 -> 12 migration, we need to create a new section for 12 -> 13.

**Review by chapulina — APPROVED** [review]
LGTM with 🟢  CI

File: Migration.md (hunk 1 above)
Comment (methylDragon) [reply Migration.md]: Rectified in force-push https://github.com/gazebosim/sdformat/pull/1022/commits/6883a17bb39d9aed8bcc3a1bd8b3ebc38731cae7

File: src/Material.cc [hunk 2]
```diff
@@ -205,61 +202,49 @@ Errors Material::Load(sdf::ElementPtr _sdf)
...
-double Material::Shininess() const
-{
-  return this->dataPtr->shininess;
-}
-
-//////////////////////////////////////////////////
-void Material::SetShininess(const double _shininess)
-{
-  this->dataPtr->shininess = _shininess;
-}
```
Comment (chapulina) [inline src/Material.cc:253]: Oops, we shouldn't remove this

**Review by chapulina — CHANGES_REQUESTED** [review]

File: src/Material.cc (hunk 2 above)
Comment (methylDragon) [reply src/Material.cc:253]: :O, how did that get removed, I think it's because SDFormat is due for a rebase?

Comment (methylDragon) [reply src/Material.cc:253]: Rebased! Change doesn't appear anymore.

Also I took the liberty of cleaning up some dangling github links from the rebase
https://github.com/gazebosim/sdformat/pull/1022/commits/8920295f0f15f7fc87411589d7208dfa205bfc6d

File: include/sdf/Material.hh [hunk 3]
```diff
@@ -59,37 +59,45 @@ namespace sdf
...
-    public: void SetSpecular(const ignition::math::Color &_color);
+    public: void SetSpecular(const gz::math::Color &_color);
+
+    /// \brief Get the specular exponent.
+    /// \return Specular exponent.
+    public: double Shininess() const;
+
+    /// \brief Set the specular exponent.
+    /// \param[in] _shininess Specular exponent.
+    public: void SetShininess(const double _shininess);
```
Comment (chapulina) [inline include/sdf/Material.hh:100]: Now these are duplicated, see the lines below

**Review by chapulina — CHANGES_REQUESTED** [review]

File: include/sdf/Material.hh (hunk 3 above)
Comment (methylDragon) [reply include/sdf/Material.hh]: My bad! Force-pushed and fixed! https://github.com/gazebosim/sdformat/compare/ff7d50728bf65bc2385580a7d5a02d7cdfaacde4..731ee8b4de53c1e89310f3bad80534893a89b43b

**Review by chapulina — APPROVED** [review]
LGTM with 🟢 CI

File: include/sdf/Cylinder.hh [hunk 4]
```diff
@@ -65,12 +65,12 @@ namespace sdf
     public: sdf::ElementPtr Element() const;
 
     /// \brief Get the Ignition Math representation of this Cylinder.
-    /// \return A const reference to an ignition::math::Cylinderd object.
-    public: const ignition::math::Cylinderd &Shape() const;
+    /// \return A const reference to an gz::math::Sphered object.
```
Comment (scpeters) [inline include/sdf/Cylinder.hh:68]: nit: revert the `Sphered` change back to `Cylinderd`

**Pull request merged**